tests/store/_unity_catalog/model_registry/test_unity_catalog_rest_store.py

```diff
@@ -114,6 +117,45 @@ def test_create_registered_model(mock_http, store):
     )
 
 
+@pytest.fixture()
+def local_model_dir(tmp_path):
+    fake_signature = ModelSignature(inputs=Schema([]), outputs=Schema([]))
+    fake_mlmodel_contents = {"signature": fake_signature.to_dict()}
+    with open(tmp_path.joinpath(MLMODEL_FILE_NAME), "w") as handle:
+        yaml.dump(fake_mlmodel_contents, handle)
+    yield tmp_path
+
+
+def test_create_model_version_missing_mlmodel(store, tmp_path):
```

Comment:
Test error cases, i.e. attempting to create a model version without an MLmodel file